src/api/auth-api.js

```diff
@@ -1,17 +1,15 @@
 import http from './http';
 
 export const signIn = async (values) => {
-  await http.post('/api/auth/signin', values)
-    .then((req) => {
-      localStorage.setItem('access-token', req.data);
-    });
+  const response = await http.post('/api/auth/signin', values);
+  localStorage.setItem('access-token', response.data);
```

Comment:
set the same as you write for signup
https://github.com/ss-ita/checklisty-web/pull/72/files#diff-e5d25ccf9ac553f2edda20e2eded5676R11